src/ctelnet.cpp

```diff
@@ -1161,26 +1161,26 @@ void cTelnet::setGMCPVariables(const QString& msg)
     QString var;
     QString arg;
     if (msg.indexOf('\n') > -1) {
-        var = msg.section("\n", 0, 0);
-        arg = msg.section("\n", 1);
+        var = msg.section(QStringLiteral("\n"), 0, 0);
+        arg = msg.section(QStringLiteral("\n"), 1);
     } else {
-        var = msg.section(" ", 0, 0);
-        arg = msg.section(" ", 1);
+        var = msg.section(QStringLiteral(" "), 0, 0);
+        arg = msg.section(QStringLiteral(" "), 1);
     }
 
-    if (msg.startsWith("Client.GUI")) {
+    if (msg.startsWith(QStringLiteral("Client.GUI"))) {
         if (!mpHost->mAcceptServerGUI) {
             return;
         }
 
         QString version = msg.section('\n', 0);
-        version.replace("Client.GUI ", "");
-        version.replace('\n', " ");
+        version.replace(QStringLiteral("Client.GUI "), QString());
+        version.replace('\n', QStringLiteral(" "));
         version = version.section(' ', 0, 0);
 
         int newVersion = version.toInt();
         if (mpHost->mServerGUI_Package_version != newVersion) {
-            QString _smsg = QString("<The server wants to upgrade the GUI to new version '%1'. Uninstalling old version '%2'>")
+            QString _smsg = QStringLiteral("<The server wants to upgrade the GUI to new version '%1'. Uninstalling old version '%2'>")
```

Comment:
Not sure about `<`...`>` in the messages - could be changed to a `[ INFO ] -` one that gets sent via `postMessage(...)` IMO.

Reply:
Not looking to make stylistic changes atm

Reply:
Okay, I can accept the need not to unnecessary changes to the text - however I think this is a `tr(`...`)` {as it is a GUI text} rather than a `QStringLiteral(`...`)`!

Reply:
Updated.